queries.sql

```diff
@@ -23,3 +23,57 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 
 /*Animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)*/
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Update and revert species column
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] Let's verify that change was made and rollbacked.
`Begin`
  update...

  Select....// to verify if change was made
`ROLLBACK`;
  Select...// to verify that change was undone